utopia-remix/app/env.server.ts

```diff
@@ -8,8 +8,10 @@ declare global {
 
 export type AppEnv = 'local' | 'stage' | 'prod' | 'test'
 
+const serverEnvironemnt: AppEnv = mustEnv('APP_ENV') as AppEnv
```

Comment:
?

Reply:
I just extracted it, since later calls to`mustEnv` inside the `ServerEnvironment` object need the server environment variable, so it can't be a key in the object initializtion